crm_autoalias/README.rst

```diff
@@ -0,0 +1,32 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+CRM: Autoalias
+======================
+
+Features
+========
+
+* Creates an email alias automatically when creating a new lead/opportunity for attach the emails
+sent to that alias.
```

Comment:
What is the expression/sequence to use for the automatic alias? If it's a sequence, put it on the Configuration section what's its name.